docs/kf.dev/content/en/docs/developers/reference-apps/awwvision/_index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Awwvision App "
+linkTitle: "Awwvision App"
+weight: 20
+description: >
+  Learn how to deploy the Awwvision reference application originally built
+  for Cloud Foundry and the Google Cloud service broker.
+---
+
+[cloud-vision]: https://cloud.google.com/vision/
+[aww]: https://reddit.com/r/aww
+[gcs]: https://cloud.google.com/storage/
+Awwvision is a Spring Boot application that uses the [Google Cloud Vision API][cloud-vision] to classify
+images from Reddit's [/r/aww][aww] subreddit, store the images and classifications in [Google Cloud
+Storage (GCS)][gcs], and display the labeled results in a web app. It uses the GCP Service Broker to
+authenticate to the Vision and Storage APIs.
+
+[original]: https://github.com/GoogleCloudPlatform/cloud-vision/tree/master/python/awwvision
+Awwvision was inspired by the [original Python version][original].
+
+{{% alert title="Costs Money" color="warning" %}}
+Following these instructions will result in creation and use of a GCS bucket
+that will incur billing charges on your GCP account.
+{{% /alert %}}
+
+## Prerequisites
+
+[google-broker]: /docs/operators/service-brokers/google-cloud/
+1. A Kf cluster and the `kf` CLI installed on your workstation
```

Comment:
Can we link to install instructions for kf?

Reply:
Done